package.json

```diff
@@ -4,10 +4,10 @@
   "description": "The javascript library client that can be used to manage decentralised identities on energy web chains",
   "main": "index.js",
   "scripts": {
-    "test": "lerna run test --stream",
+    "test": "lerna run test",
     "test-rpc": "run-with-testrpc -m \"candy maple cake sugar pudding cream honey rich smooth crumble sweet treat\" --port 8544 --accounts 20 --networkId=9 --gasLimit=10000000 \"lerna run test\" ",
     "setup": "lerna bootstrap --hoist --no-ci",
-    "build": "rm -rf build docs packages/*/dist && lerna run compile",
+    "build": "rm -rf build docs packages/*/dist && lerna run compile && lerna run build",
```

Comment:
I think details of the build process should be contained in packages. Besides it is not clear from the first look what is difference between `build` and `compile`

Reply:
I guess learn works on the repository level and to enable `typechain` build for `RevocationRegistry` I added  it to the main `package.json` to invoke the build script call.

Reply:
Moved the build inside package!